test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/liability/ExpectToExceedThresholdWeightDateControllerSpec.scala

```diff
@@ -108,16 +108,25 @@ class ExpectToExceedThresholdWeightDateControllerSpec extends ControllerSpec {
     }
 
     "redisplay page when invalid date supplied" in {
-      val resp = controller.submit()(postRequestEncoded(Date(Some(1), None, None)))
+      val resp = controller.submit()(postRequestEncoded(Date(LocalDate.of(2022, 7, 1))))
 
       status(resp) mustBe BAD_REQUEST
       contentAsString(resp) mustBe "Expect to exceed threshold date"
     }
 
     "update registration and redirect when valid date supplied" in {
       mockRegistrationUpdate()
-      val dateRealised = Date(Some(1), Some(5), Some(2022))
-      val resp         = await(controller.submit()(postRequestEncoded(dateRealised)))
+      val dateRealised = Date(LocalDate.of(2022, 4, 1))
+      val resp = await(
+        controller.submit()(
+          postJsonRequestEncodedFormAction(
+            Seq(("expect-to-exceed-threshold-weight-date.day", "1"),
+                ("expect-to-exceed-threshold-weight-date.month", "4"),
+                ("expect-to-exceed-threshold-weight-date.year", "2022")
```

Comment:
I think we should reference fields from `dateRealised`.

Reply:
Done